applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -267,14 +267,14 @@ class ExplicitCentralDifferencesScheme
         ProcessInfo& r_current_process_info = rModelPart.GetProcessInfo();
 
         const auto it_elem_begin = rModelPart.ElementsBegin();
-        #pragma omp parallel for
+        #pragma omp parallel for schedule(guided,512)
         for(int i=0; i<static_cast<int>(rModelPart.Elements().size()); ++i) {
             auto it_elem = it_elem_begin + i;
             it_elem->InitializeNonLinearIteration(r_current_process_info);
         }
 
         const auto it_cond_begin = rModelPart.ConditionsBegin();
-        #pragma omp parallel for
+        #pragma omp parallel for schedule(guided,512)
```

Comment:
Is this actually faster?
Did you measure?

Reply:
Indeed,yes Today @RiccardoRossi and I we reduced from 33 s to 8 s